app/src/test/java/com/codesoom/assignment/domain/CatToyRegistoryTest.java

```diff
@@ -0,0 +1,119 @@
+package com.codesoom.assignment.domain;
+
+import com.codesoom.assignment.dto.CatToyNotFoundException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("CatToyRegistoryTest 클래스")
+class CatToyRegistoryTest {
+
+    private CatToyRegistory catToyRegistory;
+    private CatToy catToy1;
+    private CatToy catToy2;
+    private CatToy catToy3;
+
+    @BeforeEach
+    void setUp() {
+        catToy1 = new CatToy(1L, "toy1", "maker1", 1000L, "toy1.jpg");
+        catToy2 = new CatToy(1L, "toy1", "maker1", 1000L, "toy1.jpg");
+        catToy3 = new CatToy(1L, "toy1", "maker1", 1000L, "toy1.jpg");
+
+        catToyRegistory = new CatToyRegistory();
+    }
+
+    @Nested
+    @DisplayName("create 메소드")
+    class Describe_create {
+
+        @Test
+        @DisplayName("catToy가 추가됩니다.")
+        void it_add_catToy() {
+            assertThat(catToyRegistory.getAll()).hasSize(0);
+            catToyRegistory.create(catToy1);
+            assertThat(catToyRegistory.getAll()).hasSize(1);
+        }
+    }
+
+    @Nested
+    @DisplayName("getAll 메소드")
+    class Describe_getAll {
+
+        @Nested
+        @DisplayName("catToy 데이터가 없다면")
+        class Context_has_no_catToy {
+
+            @Test
+            @DisplayName("빈 데이터를 반환합니다.")
+            void it_return_empty_catToys() {
+                List<CatToy> catToys = catToyRegistory.getAll();
+
+                assertThat(catToys).hasSize(0);
+            }
+        }
+
+        @Nested
+        @DisplayName("catToy 데이터가 있다면")
+        class Context_has_catToy {
+
+            @BeforeEach
+            void prepare() {
+                catToyRegistory.create(catToy1);
+                catToyRegistory.create(catToy2);
+                catToyRegistory.create(catToy3);
+            }
+
+            @Test
+            @DisplayName("모든 catToy를 반환합니다.")
+            void it_return_all_catToy() {
+                List<CatToy> catToys = catToyRegistory.getAll();
+
+                assertThat(catToys).hasSize(3);
+                assertThat(catToys.get(0)).isEqualTo(catToy1);
+                assertThat(catToys.get(1)).isEqualTo(catToy2);
+                assertThat(catToys.get(2)).isEqualTo(catToy3);
+            }
+        }
```

Comment:
테스트 코드에서도 하드코딩을 하지 않는 것은 중요합니다. for와 List 타입을 활용해 보세요. 다음과 같은 방법을 생각할 수 있겠죠.
```suggestion
        @Nested
        @DisplayName("catToy 데이터가 있다면")
        class Context_has_catToy {
            final List<CatToy> givenCatToys = List.of(catToy1, catToy2, catToy3);

            @BeforeEach
            void prepare() {
                for (CatToy toy: givenCatToys) {
                    catToyRegistory.create(toy);
                }
            }

            @Test
            @DisplayName("모든 catToy를 반환합니다.")
            void it_return_all_catToy() {
                List<CatToy> catToys = catToyRegistory.getAll();

                assertThat(catToys).hasSize(givenCatToys.size());
                ...
            }
        }
```

Reply:
너무 하드코딩 했네요. 감사합니다.